docs/reference/objects/product/modifier.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: default
+title: Modifier
+parent: Product
+---
+
+When using a `modifier` object you have access to the following attributes:
+
+# modifier.id
+
+Returns a unique id for the modifier.
+
+# modifier.checkbox_attributes
+
+Returns a string of HTML attributes to be used on a checkbox for the modifier.
+
+{% raw %}
+```
+<input {{modifier.checkbox_attributes}}>
+```
+{% endraw %}
+
+# modifier.contents
```

Comment:
Make me think if this is the right name for this method, anyway out of scope of this.